clang/test/CodeGen/bit-int-ubsan.c

```diff
@@ -1,5 +1,5 @@
 // REQUIRES: x86-registered-target
```

Comment:
Is this REQUIRED needed? You normally do not need the backend to be available for clang codegen tests.

Reply:
It present in other tests which are target specific. Anyways this test will not work properly for targets big endian. So it is required.